python_modules/dagster-graphql/dagster_graphql/schema/external.py

```diff
@@ -146,12 +146,18 @@ class GrapheneWorkspaceLocationStatusEntry(graphene.ObjectType):
     loadStatus = graphene.NonNull(GrapheneRepositoryLocationLoadStatus)
     updateTimestamp = graphene.NonNull(graphene.Float)
 
+    permissions = graphene.Field(non_null_list(GraphenePermission))
+
     class Meta:
         name = "WorkspaceLocationStatusEntry"
 
     def __init__(self, id, name, load_status, update_timestamp):
         super().__init__(id=id, name=name, loadStatus=load_status, updateTimestamp=update_timestamp)
 
+    def resolve_permissions(self, graphene_info):
+        permissions = graphene_info.context.permissions_for_location(location_name=self.name)
+        return [GraphenePermission(permission, value) for permission, value in permissions.items()]
```

Comment:
should i deprecate or remove the other one?

Reply:
I could imagine a world in which the current one is being accessed, I have seen people asking about programatic access to the users page in cloud for example